packages/component-base/src/overlay-class-mixin.js

```diff
@@ -69,7 +69,7 @@ export const OverlayClassMixin = (superclass) =>
       }
 
       // Add new classes based on the overlayClass
-      const classesToAdd = typeof overlayClass === 'string' ? overlayClass.split(' ') : [];
+      const classesToAdd = typeof overlayClass === 'string' ? overlayClass.split(' ').filter(Boolean) : [];
```

Comment:
The reason of the problem is that `''.split(' ')` produces an array `['']` so we need to filter out empty string.